libtransmission/verify.cc

```diff
@@ -233,29 +166,25 @@ void tr_verify_worker::add(tr_torrent* tor)
     }
 }
 
-void tr_verify_worker::remove(tr_torrent* tor)
+void tr_verify_worker::remove(tr_sha1_digest_t const& info_hash)
 {
-    TR_ASSERT(tr_isTorrent(tor));
-
     auto lock = std::unique_lock(verify_mutex_);
```

Comment:
```suggestion
    auto lock = std::lock_guard{ verify_mutex_ };
```

Reply:
can't do this; we need a `unique_lock<>` for the call to `cv.wait()` a few lines below